src/detect-mark.c

```diff
@@ -253,81 +253,65 @@ static int DetectMarkPacket(DetectEngineThreadCtx *det_ctx, Packet *p,
 /**
  * \test MarkTestParse01 is a test for a valid mark value
  *
- *  \retval 1 on succces
- *  \retval 0 on failure
```

Comment:
Not sure if Victor meant to get this removed. I think he was referring to the fact that FAIL macro means return 0 and PASS means return 1 anyway.
@victorjulien ?

Reply:
No I think its fine to remove them.

Reply:
I prefer them to be removed even.